ui/packages/consul-ui/app/services/repository/service-instance.js

```diff
@@ -1,11 +1,15 @@
 import RepositoryService from 'consul-ui/services/repository';
+import { inject as service } from '@ember/service';
+import { set, get } from '@ember/object';
+
 const modelName = 'service-instance';
 export default class ServiceInstanceService extends RepositoryService {
+  @service('repository/proxy') proxyRepo;
   getModelName() {
     return modelName;
   }
 
-  findByService(slug, dc, nspace, configuration = {}) {
+  async findByService(slug, dc, nspace, configuration = {}) {
```

Comment:
are we using the `async` here?

Reply:
Yeah, we pretty much went to 'we are an ES6 project now' with that native class codemod we did, so I'm trying to add `async` style to things as I go. If I happen to be working on something else and need to touch a file, I'll update what is straightforwards to async, unless there's a specific reason not to.